test/ngMock/angular-mocksSpec.js

```diff
@@ -351,6 +351,18 @@ describe('ngMock', function() {
     }));
 
 
+    it('should allow you to NOT specify the delay time', inject(function($interval) {
+      var counter = 0;
+      $interval(function() {counter++;});
+      $interval(function() {counter++;}, 0);
+
+      $interval.flush(1000);
+      expect(counter).toBe(2000);
```

Comment:
Duh! Both callbacks increment the same counter.
Still it might be worth having two separate counters to be safer.